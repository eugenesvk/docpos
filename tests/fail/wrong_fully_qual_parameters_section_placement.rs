/// here are some comments
/// this arguments section should not be here
#[docpos::parameters_section]
/// and some more
#[docpos::roxygen]
pub fn add(
    /// some comments
    first: i32,
    second: i32,
) -> i32 {
    first + second
}

pub fn main() {}
