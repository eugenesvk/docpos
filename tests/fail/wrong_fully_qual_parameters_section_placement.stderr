error: The #[roxygen] attribute must come before the parameters_section attribute.
       Place it before any of the doc comments for the function.
 --> tests/fail/wrong_fully_qual_parameters_section_placement.rs:5:1
  |
5 | #[docpos::roxygen]
  | ^^^^^^^^^^^^^^^^^^
